calour/heatmap/heatmap.py

```diff
@@ -252,17 +250,114 @@ def format_coord(x, y):
         else:
             return 'x=%1.2f, y=%1.2f' % (x, y)
     ax.format_coord = format_coord
+    return fig
 
-    fig.tight_layout()
 
-    return fig
+def ax_color_bar(axis, values, width, position=0, colors=None, horizontal=True):
```

Comment:
do we want to us axis for the sample/feature (instead of horizontal), and change the current axis to figure_axis?